### DISCUSSION TITLE
关于 shanwenxiao zhuchenrui chenlangning 性别的讨论

### DISCUSSION THREAD

**Discussion by zhouyiqing0304** [XMOJ-Script-dev/XMOJ-Script#432] [General]
### 检查项

- [X] 我已确认了XMOJ增强脚本已为最新版，且最新版未实现这一功能。
- [X] 我已在 [Issues 页面](https://github.com/XMOJ-Script-dev/XMOJ-Script/issues) 中搜索，确认了这一提案未被提交过。

### 描述

高老师的OJ                                                                      ▤ 
竞赛
题库
......
---------------------------------------------------------------
点这里可以关掉上面的菜单
O

**Comment by PythonSmall-Q** [comment]
邮箱提供一下

**Comment by zhouyiqing0304** [comment]
***
@PythonSmall-Q

**Comment by PythonSmall-Q** [comment]
Sent

**Comment by zhouyiqing0304** [comment]
我看看

**Comment by PythonSmall-Q** [comment]
https://dash.cloudflare.com/89969bdf9d5ab8202f8ad8b8ae2c40b8/workers/d1/databases/f6607070-7f3b-4a9b-95a2-40ff67283362
在D1里面，别乱改数据 @zhouyiqing0304

**Comment by PythonSmall-Q** [comment]
@zhouyiqing0304 succeeded？

**Comment by zhouyiqing0304** [comment]
似乎可以,我不会乱改数据的.把刚刚带链接的删掉,别泄露出去了

**Comment by langningchen** [comment]
什么？

**Comment by langningchen** [comment]
他是不会泄露出去。

**Comment by langningchen** [comment]
其他人有了链接也没有权限 ~就像我~

**Comment by PythonSmall-Q** [comment]
没事的，反正他们上去了也要登陆许可账户的

**Comment by zhouyiqing0304** [comment]
我在这里看到了一堆东西,但是不知道怎么搞.标称怎么看啊https://dash.cloudflare.com/89969bdf9d5ab8202f8ad8b8ae2c40b8/workers/d1/databases/e6ebecc6-f349-4368-9876-faa6b2376d79/tables/file_chunk

**Comment by PythonSmall-Q** [comment]
std_code

**Comment by PythonSmall-Q** [comment]
@zhouyiqing0304  看到review先别急着merge，看看没问题了就accept，等全部accept了再看看要不要merge

**Comment by PythonSmall-Q** [comment]
@zhouyiqing0304 是xmoj-bbs那个数据库不是xmoj-files

**Comment by PythonSmall-Q** [comment]
你看错库了

**Comment by zhouyiqing0304** [comment]
o

**Comment by zhouyiqing0304** [comment]
那要怎么搞

**Comment by zhouyiqing0304** [comment]
std_answer?

**Comment by PythonSmall-Q** [comment]
嗯

**Comment by zhouyiqing0304** [comment]
我要怎么在这里面搜索:https://dash.cloudflare.com/89969bdf9d5ab8202f8ad8b8ae2c40b8/workers/d1/databases/f6607070-7f3b-4a9b-95a2-40ff67283362/tables/std_answer

**Comment by PythonSmall-Q** [comment]
没办法搜索，要自己翻

你也可以问一下boomzero有没有什么便捷办法

**Comment by zhouyiqing0304** [comment]
@boomzero 我要怎么在这里面搜索:https://dash.cloudflare.com/89969bdf9d5ab8202f8ad8b8ae2c40b8/workers/d1/databases/f6607070-7f3b-4a9b-95a2-40ff67283362/tables/std_answer

**Comment by zhouyiqing0304** [comment]
@PythonSmall-Q 换行缩进怎么搞

**Comment by zhouyiqing0304** [comment]
@PythonSmall-Q 我要把这个AC的东西下载下来

**Comment by PythonSmall-Q** [comment]
哦对，@zhouyiqing0304 ，不要相信 @langningchen 的话，我们三个人都是男孩纸，尽管我profile写的是she/her

**Comment by PythonSmall-Q** [comment]
> @PythonSmall-Q 换行缩进怎么搞

@zhouyiqing0304 没看懂啥意思

**Comment by zhouyiqing0304** [comment]
我也是啊

**Comment by zhouyiqing0304** [comment]
单文骁你也太邪恶了吧,[不要相信 @langningchen 的话]是人能干出来的事吗

> 哦对，@zhouyiqing0304 ，不要相信 @langningchen 的话，我们三个人都是男孩纸，尽管我profile写的是she/her

**Comment by PythonSmall-Q** [comment]
@zhouyiqing0304 去xmoj的用户设置里面设置一下自己的badge

**Comment by zhouyiqing0304** [comment]
@PythonSmall-Q 怎么搞啊,怎么设置badge

**Comment by PythonSmall-Q** [comment]
![image](https://github.com/XMOJ-Script-dev/XMOJ-Script/assets/106425289/2b0d8583-fc0b-464c-9c74-63c1617807b8)
![image](https://github.com/XMOJ-Script-dev/XMOJ-Script/assets/106425289/035b4eaa-7e54-4f54-be08-2b5c36ead4fc)

紫色底色代表管理，上面的文字自己设置

**Comment by zhouyiqing0304** [comment]
然后呢

**Comment by PythonSmall-Q** [comment]
然后点击修改就可以了

**Comment by PythonSmall-Q** [comment]
@zhouyiqing0304 管理员权限部署了，有问题@我或者boomzero

**Comment by PythonSmall-Q** [comment]
> @boomzero 我要怎么在这里面搜索:https://dash.cloudflare.com/89969bdf9d5ab8202f8ad8b8ae2c40b8/workers/d1/databases/f6607070-7f3b-4a9b-95a2-40ff67283362/tables/std_answer

@boomzero

**Comment by boomzero** [comment]
自己写SQL

**Comment by PythonSmall-Q** [comment]
> 自己写SQL
> […](#)
> On Feb 2, 2024, at 17:49, Shan Wenxiao ***@***.***> wrote: @boomzero <https://github.com/boomzero> 我要怎么在这里面搜索:https://dash.cloudflare.com/89969bdf9d5ab8202f8ad8b8ae2c40b8/workers/d1/databases/f6607070-7f3b-4a9b-95a2-40ff67283362/tables/std_answer @boomzero <https://github.com/boomzero> — Reply to this email directly, view it on GitHub <[#423 (comment)](https://github.com/XMOJ-Script-dev/XMOJ-Script/issues/423#issuecomment-1923447547)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AULMJZPCHGRGZMCZ6PUPVHTYRSZDDAVCNFSM6AAAAABCWGL23GVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMYTSMRTGQ2DONJUG4>. You are receiving this because you were mentioned.

@zhouyiqing0304

**Comment by boomzero** [comment]
@zhouyiqing0304 if a issue requires code changes, you need to @ me ~因为swx太懒不会写代码~

**Comment by zhouyiqing0304** [comment]
好的，我先close了

**Comment by langningchen** [comment]
> 哦对，@zhouyiqing0304 ，不要相信 @langningchen 的话，我们三个人都是男孩纸，尽管我profile写的是she/her

@PythonSmall-Q :angry:

**Comment by boomzero** [comment]
No, swx definitely  is a girl

**Comment by langningchen** [comment]
> No, swx definitely is a girl

And you, too! @boomzero

**Comment by boomzero** [comment]
> > No, swx definitely is a girl
> 
> And you, either! @boomzero

too, not either

**Comment by PythonSmall-Q** [comment]
别讨论我是不是女孩子了喵呜，这里是issue不是discussion。有相关的问题讨论请到discussion

**Comment by boomzero** [comment]
@PythonSmall-Q 如你所愿

**Comment by PythonSmall-Q** [comment]
那你原来的issue还要解决的呀！

**Comment by PythonSmall-Q** [comment]
快去新家issue @boomzero

**Comment by boomzero** [comment]
原来的issue是什么, 我没看懂

**Comment by langningchen** [comment]
所以还要讨论吗？ @boomzero @PythonSmall-Q

**Comment by boomzero** [comment]
Of course!

**Comment by langningchen** [comment]
@boomzero 那你自己来说两句

**Comment by PythonSmall-Q** [comment]
继续

**Comment by langningchen** [comment]
@PythonSmall-Q 你为自己说几句？

**Comment by PythonSmall-Q** [comment]
我随便男女都可以 @langningchen

**Comment by langningchen** [comment]
> 我随便男女都可以 @langningchen

好的，那你的？ @boomzero
